velox/exec/fuzzer/AggregationFuzzerBase.h

```diff
@@ -53,6 +53,24 @@ using facebook::velox::fuzzer::SignatureTemplate;
 
 constexpr const std::string_view kPlanNodeFileName = "plan_nodes";
 
+// For come functions, velox supports NaN, Infinity better than presto query
+// runner, which makes the comparison impossible.
+// Add data constraint in vector fuzzer to enforce to not generate such data for
+// those functions before they are fixed in presto query runner
+static const std::unordered_map<std::string, Constraint> functionDataConstrain =
+    {
+        {"regr_avgx", Constraint{true, true}},
+        {"regr_avgy", Constraint{true, true}},
+        {"regr_r2", Constraint{true, true}},
+        {"regr_sxx", Constraint{true, true}},
+        {"regr_syy", Constraint{true, true}},
+        {"regr_sxy", Constraint{true, true}},
+        {"regr_slope", Constraint{true, true}},
+        {"regr_replacement", Constraint{true, true}},
+        {"covar_pop", Constraint{true, true}},
+        {"covar_samp", Constraint{true, true}},
+};
```

Comment:
Have you seen functions that only allow one of NaN and Inf? I wonder whether we could combine Constraint into one bool to control both since all functions here exclude both NaN and Inf.

Reply:
Yes, for example covar_pop would have mismatch for INFINITY https://github.com/facebookincubator/velox/issues/10627